### PR TITLE
Update changelog to add missed changelog

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,399 @@ This changelog goes through the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.com).
 
+## [1.22.4](https://github.com/go-gitea/gitea/releases/tag/1.22.4) - 2024-11-14
```
Comment (Zettat123) [inline CHANGELOG.md]: Should be `https://github.com/go-gitea/gitea/releases/tag/v1.22.4` ?

**Comment by wxiaoguang** [comment]
Should we still maintain the changelog in this repo? 

These information has been duplicated in many places: commit history, github release, blog.

Maybe using github or gitea's wiki page to maintain the changelog is better and easier?

**Comment by lunny** [comment]
> Should we still maintain the changelog in this repo?
> 
> These information has been duplicated in many places: commit history, github release, blog.
> 
> Maybe using github or gitea's wiki page to maintain the changelog is better and easier?

I think some people would like to view the changelogs in the same repository. Maybe we can just store changelogs in the main branch and will not store any release/* branch.

File: CHANGELOG.md (hunk 1 above)
Comment (lunny) [reply CHANGELOG.md]: cde1b390be

**Comment by wxiaoguang** [comment]
> I think some people would like to view the changelogs in the same repository.

Who? The changelog is out-of-data for almost one year, nobody ever notices.

**Comment by silverwind** [comment]
Changelog in the GitHub releases only may be enough. As I see it, it's duplicate information. Of course this presents a certain locking into the GitHub platform, but I assume GitHub releases could be migrated.

**Review by delvh — APPROVED** [review]
I'm fine either way, regardless of our decision (keeping the CHANGELOG file only here, on all branches, only in the releases, …).

**Review by appleboy — APPROVED** [review]

**Review by yp05327 — APPROVED** [review]

**Comment by yp05327** [comment]
Maybe just giving a link to the release page is enough.

**Pull request merged**